source/exploring_catalog_and_datasets/01_looking_up_the_catalog/catalog.rst

```diff
@@ -1,79 +1,179 @@
-Looking up the catalog
+Navigating the catalog
 ======================
 
-OpenDataSoft lets you find datasets with a rich search based experience. The catalog is often accessible via the **Explore** or **Datasets** link on the header menu. It allows to filter through all the portal's datasets by keyword, publisher, date or theme, among others.
+The dataset catalog of an OpenDataSoft portal is accessible from the front office for all users allowed on the portal (i.e. for open portals it could be anyone wheareas for private portals it should be only authenticated users).
 
-.. image:: images/explore_all.png
-   :alt: Explore catalog
+The catalog, depending on the portal, can be the default landing page of that portal. However, if that portal has another page as default landing page (e.g. a homepage), a link to the catalog should be in the navigation bar at the very top of the front office. Since this navigation bar is fully customizable, the label of the link to access the catalog can be different from one portal to another (very often, "Data", "Catalog", "Explore" or "Datasets" used as labels).
 
-Navigating in the catalog
--------------------------
+The structure of a dataset catalog however remains the same no matter the portal.
 
-The main part of the catalog shows a series of cards representing each dataset available on the platform. On each card, the dataset's metadata are displayed, along with quick links to visualize the data.
+.. image:: images/catalog_fullinterface.png
+   :alt: Full interface of the catalog, with both sides
 
-.. image:: images/explore_card.png
-   :alt: Dataset card
+On the left side (1), are displayed:
 
-On this example, the card is split in two parts. On the left part, there is a theme pictogram, the title, a description, the language and license, keywords and the original publisher. On the right part, there are links to access the data table, a map and analysis tools, and to export or access the API of the dataset.
+- the number of datasets currently displayed,
```

Comment:
What does "currently displayed" mean? Maybe we should be less specific here and say something like "the number of available datasets"?